CHANGES_UNRELEASED.md

```diff
@@ -18,3 +18,16 @@ Use the template below to make assigning a version number during the release cut
   - Description of the change with a link to the pull request ([#0000](https://github.com/mozilla/application-services/pull/0000))
 
 -->
+
+## Sync Manager
+
+### What's Changed
+  - Sync manager methods have now been standardized to support all available
+    engines.  Before each method supported its own subset of the engines
+    (For example, `wipe()` and `reset()` didn't support tabs).
+
+### ⚠️ Breaking Changes ⚠️
+  - The `SyncManager.wipe()` and `SyncManager.reset()`, now expects "passwords"
```

Comment:
TBH I wouldn't call this out either - you just accidentally fixed a bug :) And it's not a breaking change to consumers here - it's kinda more like a bugfix that affects other *devices*. So I'm not sure this patch needs any README changes at all (but I'm fine with whatever you prefer)